tests/manual-test-cases/Group5-Functional-Tests/5-15-NFS-Datastore.robot

```diff
@@ -22,9 +22,10 @@ Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
 NFS Datastore Setup
     Run Keyword And Ignore Error  Nimbus Cleanup  ${list}  ${false}
     ${esx1}  ${esx2}  ${esx3}  ${vc}  ${esx1-ip}  ${esx2-ip}  ${esx3-ip}  ${vc-ip}=  Create a Simple VC Cluster  datacenter1  cls1
-    Set Suite Variable  @{list}  ${esx1}  ${esx2}  ${esx3}  ${vc}
+    Set Suite Variable  @{list}  ${esx1}  ${esx2}  ${esx3}  %{NIMBUS_USER}-${vc}
```

Comment:
The VC we get back in `${esx1}  ${esx2}  ${esx3}  ${vc}  ${esx1-ip}  ${esx2-ip}  ${esx3-ip}  ${vc-ip}=  Create a Simple VC Cluster  datacenter1  cls1` is `${vc}`, are we saying `Set Suite Variable  @{list}  ${esx1}  ${esx2}  ${esx3} %{NIMBUS_USER}-${vc}` because Nimbus prepends the name with the Nimbus user?

Reply:
yep